Core/HW/StereoResampler.h

```diff
@@ -25,6 +25,7 @@
 
 #include "Common/ChunkFile.h"
 #include "Common/CommonTypes.h"
+#include "Core/WaveFile.h"
```

Comment:
Can we move this include to the files that need it?  We try to keep includes down to reduce churn and compile times.

-[Unknown]